cmd/skaffold/app/cmd/build.go

```diff
@@ -110,5 +110,13 @@ func createRunnerAndBuild(ctx context.Context, buildOut io.Writer) ([]build.Arti
 			targetArtifacts = append(targetArtifacts, artifact)
 		}
 	}
-	return runner.BuildAndTest(ctx, buildOut, targetArtifacts)
+	bRes, err := runner.BuildAndTest(ctx, buildOut, targetArtifacts)
```

Comment:
I'd argue it would be useful to have buildResults in the BuildOutput as well. For `skaffold build`'s output now errors could be captured intelligently as wel...WDYT?

Reply:
agreed, updated